ElectromagneticModels/coupledElectromagneticModels/coupledElectricPotential/coupledElectricPotentialFvPatchScalarField.C

```diff
@@ -0,0 +1,221 @@
+/*---------------------------------------------------------------------------*\
+  =========                 |
+  \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox
+   \\    /   O peration     | Website:  https://openfoam.org
+    \\  /    A nd           | Copyright (C) 2011-2023 OpenFOAM Foundation
+     \\/     M anipulation  |
+-------------------------------------------------------------------------------
+License
+    This file is part of OpenFOAM.
+
+    OpenFOAM is free software: you can redistribute it and/or modify it
+    under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    OpenFOAM is distributed in the hope that it will be useful, but WITHOUT
+    ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+    FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+    for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with OpenFOAM.  If not, see <http://www.gnu.org/licenses/>.
+
+\*---------------------------------------------------------------------------*/
+
+#include "coupledElectricPotentialFvPatchScalarField.H"
+#include "electromagneticModel.H"
+#include "volFields.H"
+#include "fvPatchFieldMapper.H"
+#include "mappedPatchBase.H"
+#include "addToRunTimeSelectionTable.H"
+
+// * * * * * * * * * * * * * Protected Member Functions  * * * * * * * * * * //
+
+void Foam::coupledElectricPotentialFvPatchScalarField::getNbr
+(
+    tmp<scalarField>& sigmaNbr,
+    tmp<scalarField>& sigmaEPotNbr
+) const
+{
+    const electromagneticModel& em =
+        patch().boundaryMesh().mesh()
+       .lookupType<electromagneticModel>();
+
+    sigmaNbr = em.sigma(patch().index());
```

Comment:
I could not tell differences of those sigmas just by names. how does sigma differ from sigmaEPot
After some thinking I understood that Nbr is probably neighbor, not number. Comment would help.